src/core.js

```diff
@@ -676,7 +676,11 @@ function tickAnimationFrame() {
     for (let i = 0; i < localCbs.length; i++) {
       const rafCb = localCbs[i];
       if (rafCb && rafCb[symbols.windowSymbol].document.hidden) {
-        rafCb(performanceNow);
+        try {
+          rafCb(performanceNow);
+        } catch (e) {
+          console.error(e);
```

Comment:
Wondering if this should be `console.warn(e);`?

With `console.error(e)` in `node` I see this line here being output, which is not as useful as the actual user error.

Reply:
Updated.